src/app.config.json

```diff
@@ -283,7 +280,10 @@
           "selector": "check-list",
           "settings": {
             "options": [
-              { "name": "SEARCH.CATEGORIES.SIZE_OPTIONS.SMALL", "value": "content.size:[0 TO 1048576>" },
+              {
+                "name": "SEARCH.CATEGORIES.SIZE_OPTIONS.SMALL",
+                "value": "content.size:[0 TO 1048576>"
```

Comment:
missing closing `]` braket

Reply:
is it needed? it's not a new change

Reply:
yeah.. that was used to define the inclusive start of the range.